examples/api/test/material/icon_button/icon_button.3_test.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/icon_button/icon_button.3.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('It should select and unselect the icon buttons', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.IconButtonToggleApp(),
+    );
+    expect(find.widgetWithIcon(IconButton, Icons.settings_outlined), findsExactly(8));
```

Comment:
```suggestion
    );

    expect(find.widgetWithIcon(IconButton, Icons.settings_outlined), findsExactly(8));
```

Reply:
Add extra line between pumpWidget and expect.

Reply:
I added it in [format: Better formatting](https://github.com/flutter/flutter/pull/149988/commits/543e262b3df9fd24c856e356c055cfa433738c8b)